fhir-benchmark/src/main/java/com/ibm/fhir/benchmark/FHIRParserBenchmark.java

```diff
@@ -31,52 +32,63 @@
         FHIRParser jsonParser = FHIRParser.parser(Format.JSON);
         FHIRParser xmlParser = FHIRParser.parser(Format.XML);
     }
-    
+
     @State(Scope.Benchmark)
     public static class FHIRParserState {
+        public static final String SPEC_EXAMPLE_NAME = System.getProperty(PROPERTY_EXAMPLE_NAME);
```

Comment:
does using a system property here instead of a JMH parameter limit us in any way?

Reply:
This is how we do it in several of the other benchmarks (e.g. `FHIRParserValidatorGeneratorBenchmark`) and we haven't encountered any issues prior.